### PR TITLE
Remove --binstubs from cli/install.rb

### PR DESCRIPTION
**Pull request by smlance**
- Also, remove `bundle install --binstubs` documentation from
  the config and install manpages

### CODE REVIEW

**Comment by smlance** [comment]
man/bundle-install.ronn:104 needs to be taken care of, as well.

**Comment by indirect** [comment]
I'll wait to merge this until the manpage is edited, then...

File: spec/runtime/executable_spec.rb [hunk 1]
```diff
@@ -8,142 +8,9 @@
     G
   end
```
Comment (smlance) [inline spec/runtime/executable_spec.rb]: It seems like this file was primarily testing `bundle install --binstubs`, so most of its specs have been removed.

**Comment by smlance** [comment]
Because of https://github.com/bundler/bundler/pull/3662/files, we needed to remove the first spec in `spec/commands/help_spec.rb`.

File: UPGRADING.md [hunk 2]
```diff
@@ -16,4 +16,10 @@ This change is not expected to impact production deployment scripts, because tho
...
\ No newline at end of file
+The `bundler/capistrano` and `bundler/vlad` deployment helper files have been removed. Capistrano 3 has built-in Bundler integration in the form of the `capistrano-bundler` gem, and everyone using Capistrano 3 should be using that instead. If for some reason, you are still using Capistrano 2, feel free to copy the Capistrano tasks out of the Bundler 1 file `bundler/deployment.rb` and put them into your app.
+
+#### Removed: `bundle install --binstubs`
+
+The `--binstubs` option has been removed from `bundle install` and replaced with the `bundle binstubs` command. This means that binstubs must be created and checked into version control individually.
+
+The `bundle binstubs [GEM]` command accepts two optional arguments: `--force`, which overwrites existing binstubs if they exist, and `--path=PATH`, which specifies the directory in which the binstubs will be installed (./bin by default).
```
Comment (segiddins) [inline UPGRADING.md]: place the `./bin` in backticks

File: spec/runtime/executable_spec.rb (hunk 1 above)
Comment (segiddins) [inline spec/runtime/executable_spec.rb]: van you make sure `bundle binstubs` is properly tested, and if not, change these specs to test it?

Comment (smlance) [inline spec/runtime/executable_spec.rb]: It looks like binstubs is tested in `spec/commands/binstubs_spec.rb`.

Comment (smlance) [inline spec/runtime/executable_spec.rb]: Although we could change these specs so they use `bundle binstubs` and include them, as long as they're not (too) redundant.

Comment (segiddins) [inline spec/runtime/executable_spec.rb]: :+1:

**Comment by segiddins** [comment]
@smlance try not to comment on individual commits, but rather on the PR itself?

**Comment by segiddins** [comment]
> What does the \"\" do at spec/runtime/executable_spec.rb:115? I'm unsure why this spec is failing, but perhaps this has line something to do with it.

It's intentionally trying to install to a path denoted by the empty string

**Pull request merged**